lib/tree.rb

```diff
@@ -1,25 +1,46 @@
 class NoApplesError < StandardError; end
 
-class AppleTree
-  attr_#fill_in :height, :age, :apples, :alive
+class Tree
+  attr_accessor :height, :age, :apples, :alive
 
   def initialize
+    @height = 0
+    @age = 0
+    @apples = 0
+    @alive = true
   end
 
   def age!
+    self.age += 1
+    if self.age > 5
+      self.add_apples
+    end
+    if self.age < 25
+      self.height += 1
+    elsif self.age >= 100
+      self.alive = false
+    end
   end
 
   def add_apples
+    self.apples += 1
   end
 
   def any_apples?
+    if self.apples > 0
+      return true
+    end
+    return false
```

Comment:
The value of the expression `self.apples > 0` is either `true` or `false`. So this method could be simplified to:

```ruby
def any_apples?
  return self.apples > 0
end
```

Rubyists avoid explicit calls to `return` when possible, so:
```ruby
def any_apples?
  self.apples > 0
end
```